app/helpers/submission_inputs_helper.rb

```diff
@@ -233,7 +233,7 @@ def ontology_view_of_input(ontology = @ontology)
       end
       c.container do
         content_tag(:div) do
-          render partial: "shared/ontology_picker_single", locals: { placeholder: "", field_name: "viewOf", selected: ontology.viewOf}
+          render SelectInputComponent.new(id: 'viewOfSelect', values: onts_for_select, name: 'ontology[viewOf]', selected: ontology.viewOf)
```

Comment:
from where do you get `onts_for_select` ?

Reply:
I just made get_ontologies_data helper cleaner